patches/ember-shiki@0.3.0.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/dist/components/code-block.js b/dist/components/code-block.js
+index 1b4eb37ff47f97bf032c3f63de003531fc813409..74c783ee5121a77bcdc9a9b227d1113cf11359f4 100644
+--- a/dist/components/code-block.js
++++ b/dist/components/code-block.js
+@@ -1,6 +1,6 @@
+ import { _ as _applyDecoratedDescriptor, a as _initializerDefineProperty, b as _defineProperty } from '../_rollupPluginBabelHelpers-e795903d.js';
+ import { getOwner } from '@ember/application';
+-import { service } from '@ember/service';
++import { inject as service } from '@ember/service';
```

Comment:
I patched `ember-shiki` because the syntax `import { service }` doesn't work in `ember-lts-3.28`.